config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   root "homes#index"
 
+  namespace :api do
+    namespace :v1 do
+      resources :monsters, only: [:index, :update]
```

Comment:
Let's only add actions that we're using (doesn't look like you have an update action in your controller)